gateway/handler.go

```diff
@@ -305,20 +278,6 @@ func (i *handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		}
 	}()
 
-	if i.config.Writable {
-		switch r.Method {
-		case http.MethodPost:
-			i.postHandler(w, r)
-			return
-		case http.MethodPut:
-			i.putHandler(w, r)
-			return
-		case http.MethodDelete:
-			i.deleteHandler(w, r)
-			return
-		}
-	}
```

Comment:
:information_source:  I've filled https://github.com/ipfs/kubo/issues/9622 to decide if we move this code back to kubo and do soft deprecation, or break existing users.